docs/tools/MASTG-TOOL-0105.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Corellium
+platform: generic
+source: https://corellium.com
+---
+
+Corellium is a powerful iOS and Android device virtualization platform that provides a comprehensive suite of tools for security researchers, developers, and testers. It allows users to create and manage virtual devices, perform dynamic analysis, and test applications in a controlled environment.
+
+## Overview
+
+Corellium offers a cloud-based solution that enables users to run virtualized iOS and Android devices. These virtual devices are fully functional and can be used for various purposes, including security testing, app development, and research. Corellium provides a web-based interface for managing virtual devices, as well as APIs for automation and integration with other tools.
+
+## Benefits
+
+1. **Scalability**: Corellium allows users to create multiple virtual devices, making it easy to scale testing efforts.
+2. **Accessibility**: The cloud-based platform can be accessed from anywhere, enabling remote collaboration and testing.
+3. **Isolation**: Virtual devices run in isolated environments, reducing the risk of affecting real devices or networks.
+4. **Flexibility**: Corellium supports a wide range of iOS and Android versions, allowing users to test applications on different OS versions and configurations.
+5. **Advanced Features**: Corellium provides advanced features such as kernel debugging, dynamic instrumentation, and network analysis.
+
+## Limitations
+
+1. **Cost**: Corellium is a commercial solution with a subscription-based pricing model, which may be expensive for some users.
+2. **Availability**: Corellium is primarily targeted at enterprise users and may not be accessible to individual researchers or small teams.
+3. **Learning Curve**: The platform offers a wide range of features, which may require some time to learn and master.
```

Comment:
Should we add the specific limitation for App Store / iMessage?

Reply:
And that about the camera as well. Anything else? Feel free to submit it as a suggestion